manifest.json

```diff
@@ -26,7 +26,7 @@
         "type": "cs.cm.stack.dashboard",
         "meta": [
           {
-            "path": "/#/stack-dashboard",
+            "path": "/stack-dashboard",
```

Comment:
do we have the global fullpage location ?

Reply:
that I am not sure. Pls check with @MaitriGurey1